.github/workflows/build_reusable.yml

```diff
@@ -178,6 +178,7 @@ jobs:
         with:
           name: test-reports
           path: |
+            test/traces
```

Comment:
this should probably have been put in a different artifact as this artifact is being downloaded in CI for datadog and it will probably make the download a lot slower

Reply:
good point, will make changes

Reply:
https://github.com/vercel/next.js/pull/59496